specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/examples/GetWorkspace.json

```diff
@@ -33,6 +33,15 @@
           },
           "sqlAdministratorLogin": "login",
           "managedResourceGroupName": "resourceGroup2",
+          "workspaceRepositoryConfiguration": {
+            "accountName": "myGithubAccount",
+            "collaborationBranch": "master",
```

Comment:
the list examples should also be updated

Reply:
list api shouldn’t have repoconfig